litestar/datastructures/headers.py

```diff
@@ -16,16 +17,17 @@
     Tuple,
     Union,
     cast,
+    get_args,
 )
 
 from multidict import CIMultiDict, CIMultiDictProxy, MultiMapping
-from pydantic import BaseModel, Extra, Field, ValidationError, validator
-from typing_extensions import Annotated
+from pydantic import Extra
```

Comment:
Is this still needed? It is referenced in the `Header.Config` class, but that class shouldn't exist anymore either, now that these aren't pydantic models.